examples/onnxruntime/inceptionV3/inceptionv3.py

```diff
@@ -0,0 +1,236 @@
+    latency.append(time.time() - start)
+    ort_outputs = io_binding.copy_outputs_to_cpu()[0]
+
+    #Get prediction for each item
+    for i in range(batch_size):
+        output = ort_outputs[i].copy().flatten()
+        output = softmax(output)  # this is optional
+        top5_catid = np.argsort(-output)[:topk]
+
+        for catid in top5_catid:
+            print(categories[catid], output[catid])
+
+def main():
+    flags = parse_input_args()
+
+    if flags.verbose:
+        print(flags)
+
+    if flags.verbose:
+        print("Reading in Imagenet classes")
+
+    # Read the categories
+    with open("./imagenet_classes.txt", "r") as f:
+        categories = [s.strip() for s in f.readlines()]
+
+    if flags.verbose:
+        print("Getting Exported Model from Torch")
+
+    # Export the model to ONNX
+    image_height = 299
+    image_width = 299
+    x = torch.randn(flags.batch, 3, image_height, image_width, requires_grad=True)
+    #torch_out = inception_v3(x)
+    torch.onnx.export(
+        inception_v3,  # model being run
+        x,  # model input (or a tuple for multiple inputs)
+        "inception_v3.onnx",  # where to save the model (can be a file or file-like object)
+        export_params=
+        True,  # store the trained parameter weights inside the model file
+        opset_version=14,  # the ONNX version to export the model to
+        do_constant_folding=
+        True,  # whether to execute constant folding for optimization
+        input_names=['input'],  # the model's input names
+        output_names=['output'])  # the model's output names
+
+    # Quantize the model
+    if flags.fp16:
+        if flags.verbose:
+            print("FP16 Quantization Enabled")
+        os.environ["ORT_MIGRAPHX_FP16_ENABLE"] = "1"  # Enable FP16 precision
+    else:
+        os.environ["ORT_MIGRAPHX_FP16_ENABLE"] = "0"  # Disable FP32 precision
+
+    session_ops = onnxruntime.SessionOptions()
+    if flags.verbose:
+        session_ops.log_verbosity_level = 0
+        session_ops.log_severity_level = 0
+
+    session_fp32 = onnxruntime.InferenceSession(
+        "inception_v3.onnx", providers=['MIGraphXExecutionProvider'], sess_options=session_ops)
+
+    if flags.verbose:
+        print("Preprocessing Batched Images")
+
+
+    # Preproccess and batch images
+    latency = []
+
+    if flags.verbose:
+        print("Read from dir " + flags.image_dir)
+        
+    fileList = get_image_list_in_dir(flags.image_dir)
+    
+    if flags.verbose:
+        print(fileList)    
+
+
+    #Setup input data feed
+    input_batch = torch.empty(flags.batch, 3,image_width,image_height)
+
+    batch_size = 0
+    for img in fileList:
+        filename = img  # change to your filename
+
+        if img == '':
+            break
+
+        if flags.verbose:
+            print("Preprocess: " + img)
+
+        input_image = Image.open(str(flags.image_dir + "/" + img))
+        preprocess = T.Compose([
+            T.Resize(342),
```

Comment:
Model i think takes 299x299 already. Is resize necessary ?

Reply:
Yes. its based on the requirements specified by the pytorch example docs:

https://pytorch.org/vision/main/models/generated/torchvision.models.inception_v3.html#torchvision.models.inception_v3

Reply:
Fair I can remove it.

Reply:
Done, remove crop + resize